com.github.wwmm.pulseeffects.yml

```diff
@@ -31,15 +31,6 @@ cleanup:
   - /lib/python*
   - /share/info
 
-add-extensions:
```

Comment:
I wonder why this was ever put here if pulseeffects only supports a limited set of plugins, and was bundling lsp and zamaudio anyway

Reply:
https://github.com/flathub/com.github.wwmm.pulseeffects/commit/e88e847a702f618df2419a14077d533e13a64498, well those future changes happend in easyeffects but not here, and I dont see an urgent need to "fix" this by using the extension.